lib/pf/person.pm

```diff
@@ -164,6 +169,14 @@ sub person_db_prepare {
             FROM person
             WHERE pid = ? ]);
 
+    $person_statements->{'persons_without_nodes_sql'} = get_db_handle()->prepare(
+        qq[ 
+            SELECT person.pid as pid from person 
+            LEFT JOIN node on node.pid=person.pid  
+            GROUP BY pid 
+            HAVING count(node.mac)=0;
+            ]);
+
```

Comment:
Should join the password table to avoid the additional lookup later in person_cleanup.